config/base.go

```diff
@@ -19,6 +19,11 @@ func save(filePath string, cfg interface{}) error {
 		return err
 	}
 	defer file.Close()
+
+	if err := os.Chmod(filePath, 0600); err != nil {
+		return err
+	}
```

Comment:
Instead of creating and then chmod, please consider replace `os.Create` with `os.OpenFile(filePath, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0600)`